SocialFilter/sections/specific.txt

```diff
@@ -3124,7 +3124,7 @@ blogs.oracle.com##a[class$="-socialtrigger"]
 pl.com.ua##.entry > #telegram
 7sur7.be##.sharing__list
 musicmap.info###facebook-share-button
-magadanmedia.ru##.pull-right
+prnewswire.com,magadanmedia.ru##.pull-right
```

Comment:
too wide rule, better `prnewswire.com##.social-share-circle`
and for mobile version:
`prnewswire.com##a[data-target="#social-share-mobile"]`